contracts/ReputationMiningCycle.sol

```diff
@@ -399,27 +398,32 @@ contract ReputationMiningCycle is ReputationMiningCycleStorage, PatriciaTreeProo
     // Require we've not submitted already.
     require(submission.jrhNNodes == 0, "colony-reputation-jrh-hash-already-verified");
 
-    // Get reputation root hash NNodes, which we need in both of the following checkJRHProofs
+    // Calculate how many updates we're expecting in the justification tree
     uint256 reputationRootHashNNodes = IColonyNetwork(colonyNetworkAddress).getReputationRootHashNNodes();
+    uint256 nLogEntries = reputationUpdateLog.length;
+
+    uint256 nUpdates = reputationUpdateLog[nLogEntries-1].nUpdates +
```

Comment:
Where is this being used?

Reply:
Only in the next line, so it can just be all done at once. This was refactored out of `checkJRHProof2` where it was used a little more, but not much.